dev/io.openliberty.jakarta.jsonb.internal_fat/fat/src/io/openliberty/jakarta/jsonb/internal/fat/YassonTranslationTest.java

```diff
@@ -0,0 +1,75 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.jakarta.jsonb.internal.fat;
+
+import java.util.Map;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+
+import componenttest.annotation.MinimumJavaLevel;
+import componenttest.annotation.Server;
+import componenttest.annotation.TestServlet;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+import test.yasson.translation.web.YassonTranslationTestServlet;
+
+@MinimumJavaLevel(javaLevel = 11)
+@RunWith(FATRunner.class)
+public class YassonTranslationTest extends FATServletClient {
```

Comment:
This test is only run on JSON-B 3.0.  You also fixed this for JSON-B 1.0 and 2.0.  Should this test also be run for JSON-B 1.0 and 2.0 as well?

Reply:
Took some finessing but I modified this test to repeat for EE8, EE9, and EE10. 
Thanks for the suggestion